net/transmission/Makefile

```diff
@@ -23,6 +23,12 @@ PKG_CPE_ID:=cpe:/a:transmissionbt:transmission
 PKG_FIXUP:=autoreconf
 PKG_INSTALL:=1
 PKG_BUILD_PARALLEL:=1
+PKG_CONFIG_DEPENDS:= \
+	CONFIG_LIBCURL_GNUTLS \
+	CONFIG_LIBCURL_MBEDTLS \
+	CONFIG_LIBCURL_OPENSSL \
+	CONFIG_LIBCURL_WOLFSSL \
+	CONFIG_LIBCURL_NOSSL
```

Comment:
Not sure if we want to have possibility to have transmission variant without SSL...

Reply:
libmbedtls should be used in case of LIBCURL_NOSSL see https://github.com/openwrt/packages/pull/13723/files#diff-3dc735e7e3d28d09b2f110d6d92050af27a448236d24f10ab3984074498f1fd9R43

Reply:
Those are just configuration depends. In the case of nossl and gnutls, libmbedtls is used.